# Params for drone size for planner
radius: 0.6 # meters
buffer: 1.5 # meters

# kinematic constraints
max_speed: 0.75 # m/s
max_acceleration: 1.0 # m/s^2
max_jerk: 4.0 # m/s^3

# tolerances
p_tol: 0.1
vel_tol: 0.15
accel_tol: 0.25
